apps/framework-docs/src/pages/moose/olap/external-tables.mdx

```diff
@@ -43,9 +43,10 @@ External tables allow you to connect Moose to database tables that are managed o
 
```

Comment:
when trying to explain this, i've found it helpful to use the "code as source of truth, vs infra as source of truth" language. might be helpful to put something like that in the overview section on this page? and maybe moose olap intro overall?